.github/workflows/push.yml

```
@@ -352,7 +352,9 @@ jobs:
             docker tag docker.io/moby/buildkit:buildx-stable-1 ${{vars.AWS_ECR_REPO_BASE}}/moby/buildkit:buildx-stable-1
             docker push ${{vars.AWS_ECR_REPO_BASE}}/moby/buildkit:buildx-stable-1
           fi
-
+      - name: Setup yarn
+        uses: ./.github/actions/setup-yarn
```

actionlint (1.7.4)

missing input "RUNS_ON_S3_BUCKET_CACHE" which is required by action "Setup yarn" defined at "./.github/actions/setup-yarn". all required inputs are "RUNS_ON_S3_BUCKET_CACHE"

(action)